targets/TARGET_NUVOTON/USBEndpoints_Nuvoton.h

```diff
@@ -0,0 +1,40 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2018-2018 ARM Limited
```

Comment:
please add SPDX identifier to new files

Reply:
@0xc0170 ,  Added SPDX identifier to 3 new files in commit 32c5e6b